hibernate-core/src/main/java/org/hibernate/engine/jdbc/env/internal/DefaultSchemaNameResolver.java

```diff
@@ -61,8 +61,8 @@ private SchemaNameResolver determineAppropriateResolverDelegate(Connection conne
 		}
 		catch (Exception ignore) {
 			log.debugf(
-					"Unable to use Java 1.7 Connection#getSchema : An error occurred trying to resolve the connection default schema resolver: "
-							+ ignore.getMessage() );
+					"Unable to use Java 1.7 Connection#getSchema : An error occurred trying to resolve the connection default schema resolver: %s",
+					ignore.getMessage() );
```

Comment:
This bug is more subtle. But imagine what would happen when the `ignore.getMessage()` contains `%s`. So it is still sort of bomb to explode at runtime.